src/js/dqa.jsx

```diff
@@ -21,122 +15,212 @@ var SortTypes = {
         
 var DQA = React.createClass({
     propTypes: {
-        content: React.PropTypes.string,
-        width: React.PropTypes.number,
+        aggregated: React.PropTypes.bool,
         data: React.PropTypes.object,
+        dict: React.PropTypes.object,
     },
 
     getDefaultProps: function() {
         return {
-            content: '',
-            width: 500,
+            aggregated: false,
             data: {},
+            dict: {}
         };
     },
 
     getInitialState: function() {
         return {
+            columnWidths: {
+                code: width,
+                desc: 3*width,
```

Comment:
I don't think so, at least I couldn't find a way. Width is a required parameter. Kinda annoying.